rwengine/src/objects/VehicleObject.cpp

```diff
@@ -792,7 +790,10 @@ void VehicleObject::grantOccupantRewards(CharacterObject* character) {
             || getVehicle()->vehiclename_ == "CABBIE"
             || getVehicle()->vehiclename_ == "BORGNINE") {
             // Earn $25 from taxi cabs
-            /// @todo implement this
+            if (!moneyCollected) {
```

Comment:
The collectSpecial() function already checks if the vehicle occupant reward has been granted. The ``moneyCollected`` variable is redundant.